docs/asdf/features.rst

```diff
@@ -413,6 +413,15 @@ details).
 Saving ASDF in FITS
 ===================
 
+.. note::
+
+    `~asdf.fits_embed.AsdfInFits` and `~asdf.fits_embed` are deprecated and
+    will be removed from an upcoming version of ASDF. Support for reading
+    and writing ASDF data within a FITS file has moved to the `stdatamodels`
+    package. Please see the the `stdatamodels`
+    :external+stdatamodels:doc:`asdf_in_fits` documentation for information
+    about migrating code from ASDF to `stdatamodels`.
```

Comment:
```suggestion
.. warning::

    `~asdf.fits_embed.AsdfInFits` and `~asdf.fits_embed` are deprecated and
    will be removed from an upcoming version of ASDF. Support for reading
    and writing ASDF data within a FITS file has moved to the `stdatamodels`
    package. Please see the the `stdatamodels`
    :external+stdatamodels:doc:`asdf_in_fits` documentation for information
    about migrating code from ASDF to `stdatamodels`.
```

Per https://github.com/asdf-format/asdf/pull/1337#issuecomment-1433731516 this should be a warning not a note.

Reply:
> I am ok with this as a note and would prefer to have it reviewed.

Changing it to a warning makes the links unclickable. I would prefer to leave this as a note.

Reply:
We have gone to all the trouble to update `sphinx_asdf` to fix this issue. If you are fine with it being a note then why did we bother with all that? Especially since `sphinx_asdf` is something which is relatively easy to release a patch for.

Reply:
I will change it to a warning.